app/frontend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.10)
+project(ModelParser)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(Protobuf REQUIRED)
+
+set(ONNX_GENERATED_DIR "./generated") 
+
+set(ONNX_PROTO_SRCS
+    "./generated/onnx.pb.cc"
+)
+
+set(ONNX_GENERATED_HEADER 
+    "./generated/onnx.pb.h"
+) 
+
+set(SRC_FILES
+    main.cpp
+    ${ONNX_PROTO_SRCS}
+)
+
+configure_file(${CMAKE_SOURCE_DIR}/generated/yolo11x.onnx
+               ${CMAKE_CURRENT_BINARY_DIR}/generated/yolo11x.onnx
+               COPYONLY)
```

Comment:
You may need to add this to get it working:
```cmake
set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/generated/yolo11x.onnx PROPERTIES GENERATED TRUE)
```
This will mark the file as autogenerated (so we kind of promise that this file will appear later). This should let you avoid cmake configuration error at this point